Tweak.x

```diff
@@ -415,8 +416,38 @@ NSDictionary *getIdentifiers() {
 
 	// This %ctor will be called every time identityservicesd or the settings app is restarted.
 	// So we only want it to try to reinitialize stuff if it's in identityservicesd
-	if (![bundleID isEqualToString:@"com.apple.identityservicesd"])
+	if (![bundleID isEqualToString:@"com.apple.identityservicesd"]) {
+		[[NSDistributedNotificationCenter defaultCenter] addObserverForName: @"com.beeper.beepserv/updateState"
```

Comment:
Could we have the notification observer names (`com.beeper.beepserv/updateState` and `com.beeper.beepserv/requestStateUpdate`) be placed into global statics? It would just make it easier to ensure we don't cause any issues with typos and such.

Reply:
That's a good idea. I've moved the observer names, along with `kCode` etc. to a new `Constants.h` so we can easily access them from both `Tweak.x` and `State.x`. Also, they are consts now, except for `kSuiteName`, which would make the code not compile as a const.